pkg/api/licenses.go

```diff
@@ -1152,3 +1152,60 @@ func ExportLicenses(c *gin.Context) {
 	c.Header("Content-Disposition", fmt.Sprintf("attachment; filename=%s", fileName))
 	c.JSON(http.StatusOK, &licenses)
 }
+
+// GetAllLicensePreviews retrieves a list of shortnames of all licenses
+//
+//	@Summary		Get shortnames of all active licenses
+//	@Description	Get shortnames of all active licenses from the service
+//	@Id				GetAllLicensePreviews
+//	@Tags			Licenses
+//	@Accept			json
+//	@Produce		json
+//	@Param			active	query		bool	true	"Active license only"
+//	@Success		200		{object}	models.LicensePreviewResponse
+//	@Failure		400		{object}	models.LicenseError	"Invalid active value"
+//	@Failure		500		{object}	models.LicenseError	"Unable to fetch licenses"
+//	@Router			/licenses/preview [get]
+func GetAllLicensePreviews(c *gin.Context) {
+	var licenses []models.LicenseDB
+	active := c.Query("active")
+	if active == "" {
+		active = "true"
+	}
+	var parsedActive bool
+	parsedActive, err := strconv.ParseBool(active)
+	if err != nil {
+		er := models.LicenseError{
+			Status:    http.StatusBadRequest,
+			Message:   "Invalid active value",
+			Error:     fmt.Sprintf("Parsing failed for value '%s'", active),
+			Path:      c.Request.URL.Path,
+			Timestamp: time.Now().Format(time.RFC3339),
+		}
+		c.JSON(http.StatusBadRequest, er)
+		return
+	}
+	query := db.DB.Model(&models.LicenseDB{})
+	query.Where("active = ?", parsedActive)
```

Comment:
Field name is `rf_active`

https://github.com/fossology/LicenseDb/blob/7940f0a8f736b23396ec18509b5fb1c9380074f3/pkg/models/types.go#L34